scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -238,49 +238,48 @@ <h4 class="card-header">Case: {{case.display_name}}</h4>
 {% endmacro %}
 
 {% macro group_panel() %}
-  <div class="card panel-default">
-    <div data-toggle='tooltip' class="panel-heading" title="Group cases to share user variant assessments like comments, classification and dismissal. A case can be in multiple groups."><span class="fa fa-users"></span>&nbsp;Group connected cases</div>
+  <div class="card panel-default" id="case_groups">
+    <div class="panel-heading">
+      <span data-toggle='tooltip' title="Group cases to share user variant assessments like comments, classification and dismissal. A case can be in multiple groups.">
```

Comment:
I actually did this on purpose, because I didn't want the tooltip to activate when you hover over the "+ new group", since that would have been very confusing. If you really don't like it I can try to find a better solution. Ideas?

Reply:
Hm, you think so? The text kind of fits? But anyway, what happens if you add a tooltip to the "+ New group" link - would it override?

Reply:
I'll try!